helloapache/README.md

```diff
@@ -1,73 +1,70 @@
-# helloapache
+# Hello Apache
 
-This is an atomic application based on the nulecule specification. Kubernetes and native docker are currently the only supported providers. You'll need to run this from a workstation that has the atomic command.  If you wish to use the kubernetes provider, you will also need a kubectl client that can connect to a kubernetes master.
+### Description
 
-It's a single container application based on the centos/httpd image, but you can use your own.
+A helloworld example using the centos/apache container.
 
-## Option 1: Non-interactive defaults
+### Deployment
 
-Run the image. It will automatically use kubernetes as the orchestration provider.
-```
-[sudo] atomic run projectatomic/helloapache
-```
-
-Note: This option is not interactive because all params in the Nulecule file have default values.
-
-## Option 2: Unattended
+#### Deploying with Default Parameters
 
-1. Create the file `answers.conf` with these contents:
+##### Using Atomic CLI
 
-    This sets up the values for the two configurable parameters (image and hostport) and indicates that kubernetes should be the orchestration provider.
-
-        [general]
-        provider = kubernetes
-
-        [helloapache-app]
-        image = centos/httpd # optional: choose a different image
-        hostport = 80        # optional: choose a different port to expose
-1. Run the application from the current working directory
-
-        $ [sudo] atomic run projectatomic/helloapache
-        ...
-        helloapache
+```sh
+sudo atomic run projectatomic/helloapache
+```
 
+##### Using Atomic App
 
-1. As an additional experiment, remove the kubernetes pod and change the provider to 'docker' and re-run the application to see it get deployed on base docker.
+```sh
+sudo atomicapp run projectatomic/helloapache
+```
 
-## Option 3: Install and Run
+#### Deploying with User-Provided Parameters
 
-You may want to download the application, review the configuraton and parameters as specified in the Nulecule file, and edit the answerfile before running the application.
+##### Using Atomic CLI
 
-1. Download the application files using `atomic run IMAGE --mode fetch`
+```sh
+sudo atomic run projectatomic/helloapache --mode fetch --destination helloapache
+cd helloapache
+cp answers.conf.sample answers.conf
```

Comment:
only thing I might add here is a note that says that they can edit the answers.conf file now:

```
cp answers.conf.sample answers.conf # make edits to answers.conf
```